src/modules/Dashboard/state/Users/Users.thunk.ts

```diff
@@ -3,17 +3,13 @@ import { AxiosError } from 'axios';
 
 import { ApiError } from 'redux/modules';
 import { getWorkspaceRespondentsApi, GetAppletsParams } from 'api';
-
-import { RESPONDENTS_WITHOUT_LIMIT } from './Users.const';
+import { MAX_LIMIT } from 'shared/consts';
 
 export const getAllWorkspaceRespondents = createAsyncThunk(
   'users/getAllWorkspaceRespondents',
   async ({ params }: GetAppletsParams, { rejectWithValue, signal }) => {
     try {
-      return await getWorkspaceRespondentsApi(
-        { params: { ...params, limit: RESPONDENTS_WITHOUT_LIMIT } },
-        signal,
-      );
+      return await getWorkspaceRespondentsApi({ params: { ...params, limit: MAX_LIMIT } }, signal);
```

Comment:
please add a TODO to discuss more suitable solution with BA: pagination or infiniti scroll.